ISSUE_TEMPLATE.md

```diff
@@ -24,48 +39,23 @@ to reach Stable.
 
 # Progress Tracker
 
+Please, select the following checkboxes and fill with the necessary data after completing the desired steps: 
+
+Before the features freeze:
+[ ] - Primary contact
+[ ] - SIG
+[ ] - Proposal link 
+[ ] - Reviewer(s)
+[ ] - Approver
+[ ] - Target
+[ ] - Timeline / status
+
+Before the coding freeze:
+[ ] - Release note
+[ ] - Documentation
+
```

Comment:
I would like to see an `Implementation fully merged` checkbox or something like that.